CHANGELOG.md

```diff
@@ -760,7 +760,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
-[develop]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.31.0...HEAD
+[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.31.0...v3.32.0
```

Comment:
```suggestion
[develop]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...HEAD
[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.31.0...v3.32.0
```

Reply:
Same as https://github.com/bitmovin/bitmovin-player-ui/pull/417#discussion_r770684232

I think it will be done automatically when finishing the release.